### PR TITLE
icons, ui-components, storybook: Icon update

### PR DESCRIPTION
**Pull request by nathanstilwell**
# Icons, UI Component, Storybook // Icon update

After an audit of the existing icons across this repo and Cloud Console, the standard set of icons has been updated to add, deprecate, and update some of our icons.

All told, this PR adds 17 new system icons, 7 new third party icons, 4 new flags, and removes the Logo icons and components. The Logos were prohibitively large and increased the weight of our dependencies, also they were not being used anywhere. Despite the large number of files changes in this PR, the best way to review would be to read the `packages/icons/src/index.ts` file changes.

Here are some proofs of the new sets as an overview,

Screenshot 2023-02-24 at 3 38 52 PM
Screenshot 2023-03-03 at 1 19 44 PM
Screenshot 2023-02-24 at 3 39 17 PM
Screenshot 2023-02-24 at 3 39 36 PM

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @cockroachlabs/icons@0.7.0-canary.504.4326932722.0
  npm install @cockroachlabs/ui-components@0.6.0-canary.504.4326932722.0
  # or 
  yarn add @cockroachlabs/icons@0.7.0-canary.504.4326932722.0
  yarn add @cockroachlabs/ui-components@0.6.0-canary.504.4326932722.0
  ```
</details>

### CODE REVIEW

**Review by janmorales7 — APPROVED** [review]
Amazing work!! Cant wait to see these come to life in the UI 🤩

File: packages/icons/src/index.ts [hunk 1]
```diff
@@ -12,52 +12,66 @@ o888o o888o o888o o888o  888bod8P' `Y8bod8P' d888b      "888" 8""888P'
...
 import { default as Bell } from "./components/Bell";
-import { default as CancelCircleFilled } from "./components/CancelCircleFilled";
-import { default as CancelCircle } from "./components/CancelCircle";
+import { default as Billing } from "./components/Billing";
+import { default as Calendar } from "./components/Calendar";
 import { default as Cancel } from "./components/Cancel";
+import { default as CancelCircle } from "./components/CancelCircle";
+import { default as CancelCircleFilled } from "./components/CancelCircleFilled";
 import { default as CaretDown } from "./components/CaretDown";
 import { default as CaretFilledDown } from "./components/CaretFilledDown";
```
Comment (sjbarag) [inline packages/icons/src/index.ts:26]: one day we'll probably want to come back and make these `import ArrowLeft from "./components/ArrowLeft";` 😃 

Not in this PR though, there's already enough going on!

**Review by sjbarag — APPROVED** [review]
Yessssssss bye logos!

File: packages/icons/src/index.ts (hunk 1 above)
Comment (nathanstilwell) [reply packages/icons/src/index.ts:26]: What is the impact of that change?

Comment (sjbarag) [reply packages/icons/src/index.ts:26]: There's fewer characters per line, that's about it :) `import { default as Foo } from` and `import Foo from` are equivalent!

**Pull request merged**